wagtail/wagtail
# Remove the Hallo Editor - now provided as an external package

* Removal of the Wagtail Hallo editor, adds notes to the release notes
* Resolves https://github.com/wagtail/wagtail/issues/6228
* New external package (alpha) is available via https://pypi.org/project/wagtail-hallo/0.1.0a1/
* Please review the removals carefully, the goal is to remove Hallo only but keep some of the core utils such as whitelister & editorhtml intact (some may be removed in the future but not in scope of this work)

## How to test this
- Check out this branch locally
- **Important**: Delete the built static assets at `wagtail/admin/static` and then run the Wagtail build pipeline `nvm use` then `npm run build` - this is required so that validation can be done without the styles/JS provided by Wagtail and only the ones provided by the new package.
- Then install the package [as per the documentation](https://github.com/wagtail/wagtail-hallo#installing-the-hallo-editor) (to also test these instructions make sense), remember to update your bakerydemo (or other local Wagtail instance) settings to use the hallo editor
- You may need to make a new page that uses the hallo editor, here are two [example models](https://github.com/wagtail/wagtail-hallo/blob/main/wagtail_hallo/test/models.py).
